internal/impl/amqp09/integration_test.go

```diff
@@ -14,30 +16,43 @@ import (
 )
 
 func TestIntegrationAMQP09(t *testing.T) {
-	integration.CheckSkip(t)
-	t.Parallel()
+	assertQueueStateFromRabbitMQManagementAPI := func(resource *dockertest.Resource) {
```

Comment:
This closure here runs as a test cleanup (because I needed to be sure the suites had completed) but it does do some some assertion.

Is there a way that's more in keeping with the pattern to do this?